cli/cli.go

```diff
@@ -0,0 +1,69 @@
+// Package cli implements some simple shared CLI flag types.
+package cli
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	pb "github.com/bazelbuild/remote-apis/build/bazel/remote/execution/v2"
+	"github.com/peterebden/go-cli-init"
+)
+
+var log = cli.MustGetLogger()
+
+var actionRe = regexp.MustCompile("([0-9a-fA-F]+)/([0-9]+)/?")
+var shortActionRe = regexp.MustCompile("([0-9a-fA-F]+)")
+
+// An Action represents a combined hash / size pair written like
+// ff17a4efe382e245491d6a9f1ac6bf3adce454f7e4a5559a3579c3856edf1381/122
+// This is a bit more concise than passing them with flags.
+type Action struct {
+	Hash string
+	Size int
+}
+
+func (a *Action) UnmarshalFlag(in string) error {
+	matches := actionRe.FindStringSubmatch(in)
```

Comment:
if strings.contains(in, "/") would do it

Reply:
yeah but then I'd have to do this anyway to get the submatches